mongodbatlas/data_source_mongodbatlas_cloud_provider_access_setup_test.go

```diff
@@ -0,0 +1,62 @@
+package mongodbatlas
```

Comment:
This file is strange:
- It is over 2 years old
- I do not see it present in master

I imagine we removed it recently and it remained in our migration branch, but cannot find that change in master to get more context.

Reply:
It was removed here https://github.com/mongodb/terraform-provider-mongodbatlas/pull/1343/files#diff-2637f6d298e2e2a741c6133aa145689eca7a4bf08551a0826bc5f8eaf5543701. the tests were moved to the resource acceptance tests. I will delete this file. Thanks!

Reply:
I removed it